src/Menu/OrderShowMenuListener.php

```diff
@@ -22,7 +22,7 @@ public function addRefundsButton(OrderShowMenuBuilderEvent $event): void
         $menu = $event->getMenu();
         $order = $event->getOrder();
 
-        if ($this->orderRefundsListAvailabilityChecker->__invoke($order->getNumber())) {
+        if ($this->orderRefundsListAvailabilityChecker->__invoke((string) $order->getNumber())) {
```

Comment:
Can you explain in what case do you have null as an order number?

Reply:
It should not be possible to have an order without a number. I would rather throw an exception if the number is null